integration_tests/cli/model-demo.sh

```diff
@@ -112,3 +112,132 @@ check_response "$result" "\"vid\": $vid"
 check_response "$result" "\"pid\": $pid"
 check_response "$result" "\"supportUrl\": \"$supportURL\""
 echo "$result"
+
+test_divider
+
+echo "Delete Model with VID: ${vid} PID: ${pid}"
+result=$(dcld tx model delete-model --vid=$vid --pid=$pid --from=$vendor_account --yes)
+echo "$result"
+
+test_divider
+
+echo "Query non existent model"
+result=$(dcld query model get-model --vid=$vid --pid=$pid)
+check_response "$result" "Not Found"
+echo "$result"
+
+
+# Chech how to work model with vid/pid in hex format
+
+# Preperation of Actors
+
+vid_in_hex_format=0xA13
```

Comment:
I recommend moving the hex-related logic into a separate file (for example `model-demo-hex.sh`).

Reply:
Done! ✅